queries.sql

```diff
@@ -8,3 +8,113 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered='true';
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that species columns went back to the state before tranasction.
+BEGIN;
+UPDATE animals SET species = 'unspecified'
+ROLLBACK;
+
+-- Inside a transaction:
+-- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+
+UPDATE animals SET species = 'pokemon' WHERE species = null;
+
+-- COMMIT THE TRANSACTION
+COMMIT;
```

Comment:
Please add queries before and after commit to check if the changes are made.

Reply:
Hi @SajjadAhmad14, please can we hop on a Zoom call or chat on slack because I don't know what exactly I'm to do here and most of my peers didn't do this so I don't know who I can ask for help.